vm/src/stdlib/json.rs

```diff
@@ -10,6 +10,11 @@ pub fn json_dumps(obj: PyObjectRef, vm: &VirtualMachine) -> PyResult<String> {
     serde_json::to_string(&serializer).map_err(|err| vm.new_type_error(err.to_string()))
 }
 
+pub fn json_dump(obj: PyObjectRef, fs: PyObjectRef, vm: &VirtualMachine) -> PyResult {
+    let result = json_dumps(obj, vm).map_err(|err| vm.new_type_error(err.to_string()));
```

Comment:
Instead of the `unwrap`, you might consider the `?` operator like this:

```rust
let result = json_dumps(obj, vm)?;
vm.call_method(&fs, "write", vec![vm.new_str(result)])
```